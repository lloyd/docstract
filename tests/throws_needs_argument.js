/**
 * a module with a function thats throws something we don't knows.
 */

/**
 * @function throweyFunc
 * @throws
 */
