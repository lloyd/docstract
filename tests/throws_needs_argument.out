{
  "args": [
    "@throws tag requires an argument"
  ], 
  "exception_type": "<type 'exceptions.RuntimeError'>"
}
